pkg/utils/validate/login_hint_test.go

```diff
@@ -0,0 +1,35 @@
+package validate
+
+import "testing"
+
+func TestLoginHint(t *testing.T) {
```

Comment:
@PedroDBFlores , please see the following testing pattern and mimic it. https://github.com/greenpau/caddy-authorize/blob/main/pkg/shared/idp/config_test.go